src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -13,6 +13,7 @@ public class TorpedoStore {
   private double FAILURE_RATE = 0.0; //NOSONAR
 
   private int torpedoCount = 0;
+  Random generator = new Random();
```

Comment:
Might be better if it explicitly stated that this is a random generator, not just any generator (in case we want to add another kind of generator in the future)